x11rb-protocol/src/utils.rs

```diff
@@ -11,70 +11,13 @@
 
 #[cfg(all(feature = "std", unix))]
 mod raw_fd_container {
-    use std::mem::forget;
-    use std::os::unix::io::{AsRawFd, IntoRawFd, RawFd};
+    use std::os::unix::io::OwnedFd;
 
     /// A simple wrapper around RawFd that closes the fd on drop.
     ///
     /// On non-unix systems, this type is empty and does not provide
```

Comment:
Could you... uhm... do something about this doc comment? (Also in the non-unix case below)

I feel like the wording no longer matches. Perhaps this should say that this is a re-export of `std::os::unix::io::OwnedFd` on unix and an empty type everywhere else? Or something like that.

Reply:
I modified it to go into more detail